src/openrct2/rct2/RCT2.h

```diff
@@ -542,7 +542,7 @@ struct RCT2SpriteVehicle : RCT12SpriteBase
     uint16_t var_C8;
     uint16_t var_CA;
     uint8_t scream_sound_id; // 0xCC
-    uint8_t TrackSubposition;
+    VehicleTrackSubposition TrackSubposition;
```

Comment:
in general we do not change the type of the RCT2 and RCT1 headers they would stay as `uint8_t` this is because this structure should only be used for reading a writing to the file not for performing any logic and it should be validated.